### DISCUSSION TITLE
如何修改部分DatePicker周的起始日

### DISCUSSION THREAD

**Discussion by simonzfq** [ant-design/ant-design#49712] [Q&A]
`dayjs.extend(updateLocale);
dayjs.updateLocale('zh-cn', {
  weekStart: 0,
});`
以上代码是官方提供的修改周的起始日的方法，但该方法会修改全部的DatePicker的周起始日，我的需求是只想修改个别DatePicker的周起始日，官方只提示说要修改DatePicker的locale属性，但没找到具体修改周起始日的属性。

**Comment by Wxh16144** [comment]
dayjs 的修改是作用于全局的，做不了只修改部分周起始日。 唯一的方法就是自己通过  `<DatePicker />` 的 `components` 自定义选择面板
